utils/utils.jsx

```diff
@@ -207,7 +207,7 @@ export function ding() {
 }
 
 export function hasSoundOptions() {
-    return (!(UserAgent.isFirefox()) && !(UserAgent.isEdge()));
+    return (!(UserAgent.isEdge()));
```

Comment:
Could you remove these extra brackets?
```suggestion
    return !UserAgent.isEdge();
```

Reply:
+1! should be fixed in 837a547